Description: drop ruby-static feature
 rb-sys does not provide ruby-static
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-05-01
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -94,4 +94,3 @@
 embed = ["rb-sys/link-ruby"]
 friendly-api = []
 rb-sys = []
-ruby-static = ["rb-sys/ruby-static"]
